src/libtexture/imagecache.cpp

```diff
@@ -2495,6 +2495,26 @@ ImageCacheImpl::get_image_info (ImageCacheFile *file,
         *(int *)data = file->is_udim();
         return true;
     }
+    if (file->is_udim() && dataname == s_channels) {
+        // Special case -- it's ok to ask for a UDIM's channels. It'll
+        // search for a concrete file. Beware, this will cause trouble
+        // if different panels of the same UDIM scheme have different
+        // numbers of channels! Search the 10x10 files for a match, give
+        // up if not found.
+        ImageCacheFile *concretefile = NULL;
+        for (int j = 0; j < 10 && !concretefile; ++j) {
```

Comment:
UDIM by convention only goes 10 in the u direction. Is 100 in the v direction enough?